src/m2n/PointToPointCommunication.hpp

```diff
@@ -114,6 +141,25 @@ class PointToPointCommunication : public DistributedCommunication
    */
   std::vector<Mapping> _mappings;
 
+   /**
+   * @brief this data structure is used to store m2n communication information for the 1 step of 
+   *        bounding box initialization. It stores:
+   *        1. global remote process rank;
+   *        2. communication object (provides point-to-point communication routines).
+   *        3. Appropriatly sized buffer to receive elements
```

Comment:
What do you mean by this? The third member below is the request. Where is the buffer? What do you mean by element?